### ISSUE TITLE
[BUG] Error "Data will be loading" - didn't work for a few days?!

### ISSUE THREAD

**Issue by Dom78** [diegomanuel/binance-to-google-sheets#129]
When I use your demo Excel sheet or my sheet which works fine the last few weeks, I see only the Error "Data will be loading" but nothing happens. 

Is it a Bug with the Binance API request, or is there another issue? 
Is there a workaround? Do you also have an Excel-Plugin to get it to run locally without the Google cloud (IP) maybe too many request from Google Drive will be blocked by the BINANCE API?
